pipeline/azure-pipelines.yml

```diff
@@ -0,0 +1,215 @@
+variables:
```

Comment:
The pipeline is hard to follow in its current form - whitespace between the sections/tasks would make it easier to read e.g. the [sonar-secrets dotnet yaml file](https://github.com/SonarSource/sonar-secrets/blob/master/sonar-secrets-dotnet/azure-pipelines-dotnet.yml).